infrastructure/server-setup/playbook-3.yml

```diff
@@ -386,6 +386,37 @@
       copy:
         src: ../logrotate.conf
         dest: /etc/
+    
+    - name: Give read and write permissions to logrotate.conf because we replace it on each deploy
+      file:
+        path: /etc/logrotate.conf
+        owner: '{{ ansible_user }}'
+        group: '{{ ansible_user }}'
+        mode: 'u+rwX,g+rwX,o-rwx'
+    
+    - name: Create deploy logfile
+      ansible.builtin.file:
+        path: /var/log/setup-deploy-config.log
+        owner: '{{ ansible_user }}'
+        group: '{{ ansible_user }}'
+        state: touch
+        mode: 'u+rwX,g+rwX,o-rwx'
+    
+    - name: Create secret logfile
+      ansible.builtin.file:
+        path: /var/log/rotate-secrets.log
+        owner: '{{ ansible_user }}'
+        group: '{{ ansible_user }}'
+        state: touch
+        mode: 'u+rwX,g+rwX,o-rwx'
+    
+    - name: Create backup logfile
+      ansible.builtin.file:
+        path: /var/log/opencrvs-backup.log
+        owner: '{{ ansible_user }}'
+        group: '{{ ansible_user }}'
+        state: touch
+        mode: 'u+rwX,g+rwX,o-rwx'
```

Comment:
It would be best if we created a directory under /var/log/ for instance `/var/log/opencrvs` that had an owner `root` and group access for `app` group. @naftis has done this before and can advice